docs/Developers/Development_Setup.rst

```diff
@@ -20,14 +20,14 @@ you can use whichever you find suitable to yourself.
 Using venv
 ~~~~~~~~~~
 
-- Make sure to have Python 3 installed in your local machine.
+- Make sure to have Python3 installed in your local machine.
```

Comment:
you're doing this because of https://github.com/coala/coala/issues/3921#issuecomment-286710570 I presume, however in written normal text, not code samples, you usually write `Python 3`, for commands it's `python3`.